### PR TITLE
Update ngen script to work with new hydro fabric

### PR DESCRIPTION
**Pull request by donaldwj**
[Short description explaining the high-level reason for the pull request]

## Additions

-

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [ ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: src/external_connections/next_gen_network.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 from functools import partial
 from itertools import chain, islice
 import next_gen_io
+import json
```
Comment (jdmattern-noaa) [inline src/external_connections/next_gen_network.py:18]: Please change the example usage on line 3 to include the new flowpath_data.geojson file instead and remove the previous geojson file from the repo.

**Review by jdmattern-noaa — CHANGES_REQUESTED** [review]
Looks good. Please also remove the four catchment output files (87, 88, 89, and 92) that are no longer being used because the code will read all catchment files in the directory.

**Review by jdmattern-noaa — APPROVED** [review]
Looks great. Thanks @donaldwj

**Review by jameshalgren — APPROVED** [review]

**Comment by jameshalgren** [comment]
@donaldwj @jdmattern-noaa @hellkite500 
Squashed and merged this.

**Pull request merged**